src/gui/qgsbrowserdockwidget.h

```diff
@@ -102,6 +102,8 @@ class GUI_EXPORT QgsBrowserDockWidget : public QgsDockWidget, private Ui::QgsBro
     void openFile( const QString & );
     //! Emitted when drop uri list needs to be handled
     void handleDropUriList( const QgsMimeDataUtils::UriList & );
+    //! Connections changed in the browser
+    void connectionsChanged( );
```

Comment:
@elpaso The code actually comes from the WFS provider (as written at the time) and I had AFS, AMS and WFS using this base class, so not strictly for the arcgis providers. But indeed now the arcgis providers are the only ones still using it, so it might as well be moved to the arcgis provider folder and called QgsArcGisServiceSourceSelect or something similar, if it can't be used as a common base class for other online service dialogs anymore.